gen/template/test/example_test.rb

```diff
@@ -6,10 +6,12 @@ class ExampleTest < MiniTest::Test
 
     def test_example
       CLI::Kit::System.fake("ls -al", stdout: "a\nb", success: true)
-      assert_all_commands_run do
-        out, = CLI::Kit::System.capture2('ls', '-al')
-        assert_equal ['a', 'b'], out.split("\n")
-      end
+
+      out, = CLI::Kit::System.capture2('ls', '-al')
+      assert_equal %w(a b), out.split("\n")
+
+      errors = assert_all_commands_run(should_raise: false)
+      assert_nil errors, "errors should have been nil"
```

Comment:
nit: `assert_nil` itself implies the given error message. I like to make them a little more descriptive. In this case, we expected the above command to run successfully, so I'd probably have a message like "expected command to run successfully".